scripts/dura.fish

```diff
@@ -1,10 +1,10 @@
 set script (realpath (status --current-filename))
-if pgrep -fq $script
+if pgrep -f $script >/dev/null 2>/dev/null
     exit 0
 end
 
 function tempfile
-    if command_exists mktemp
+    if which mktemp >/dev/null 2>/dev/null
```

Comment:
makes more sense, i thought `command_exists` was just some fish thing. is it from powershell?

Reply:
It's a function I created such a long time ago that I forgot it isn't a builtin 😅